cli/command/container/update.go

```diff
@@ -66,6 +68,8 @@ func NewUpdateCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags.StringVar(&opts.kernelMemory, "kernel-memory", "", "Kernel memory limit")
 	flags.StringVar(&opts.restartPolicy, "restart", "", "Restart policy to apply when a container exits")
 
+	flags.Var(&opts.cpus, "cpus", "Number of CPUs")
```

Comment:
This probably requires a "API version" annotation, so that it's hidden when connected to an older daemon